routes/patient/patientEntry.js

```diff
@@ -1,37 +1,63 @@
 const router = require('express').Router();
 let PatientEntry = require('../../models/PatientEntry');
 
-//route for getting a array of all patient entries
+
 router.route('/patientEntries').get((req, res) => {
```

Comment:
this is an interesting syntax a simpler version is 'router.get('/patientEntries', (req, res) => {}